### PR TITLE
Build component page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build component page
- Update URL from model details page to point to component page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/certification/component/682 or http://0.0.0.0:8001/certification/component/710
- Check the page against the [design](https://app.zeplin.io/project/5f3f890c6afe2512594898b2/screen/6076cde233a60f47a193f604)

## Issue / Card

Fixes [#162](https://github.com/canonical-web-and-design/certification.ubuntu.com/issues/162)

## Screenshots

image

### CODE REVIEW

**Comment by anasereijo** [comment]
@bethcollins92 this is looking good! buttttt
can you add a bit more spacing between sections? see below

Screenshot 2021-04-26 at 15 39 52

**Comment by bethcollins92** [comment]
Thanks @anasereijo - made the suggested change

**Comment by anasereijo** [comment]
Thanks @bethcollins92

File: templates/certification/component-details.html [hunk 1]
```diff
@@ -1,56 +1,99 @@
...
+          <tr>
+            <th class="p-muted-text">Note</th>
+            <td>{{ component.note }}</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <h2>Present in {{ machines|length }} certified system{% if machines|length > 1%}s{% endif %}</h2>
```
Comment (carkod) [inline templates/certification/component-details.html:88]: Space between the 1 and %? 
It was weird initially I thought you were doing the condition against a percentage number, I thought how can this work?

File: templates/certification/component-details.html [hunk 2]
```diff
@@ -1,56 +1,99 @@
...
+    <ul class="p-list">
+      {% for machine in machines %}
+      <li class="p-list__item">
+        <a href="/certification/{{ machine.canonical_id }}">{{ machine.canonical_id}} {{ machine.make }} {{ machine.model }}</a>
```
Comment (carkod) [inline templates/certification/component-details.html:92]: You might as well add also a space after `machine.canonical_id`?

File: templates/certification/model-details.html [hunk 3]
```diff
@@ -133,7 +133,7 @@ <h2 class="u-sv2">Components certified with this system</h2>
       <tbody>
         {% for component in components%}
         <tr>
-          <th aria-label="Component"><a href="/certification/component/{{ canonical_id }}">{{ component.vendor_name }} {{ component.model}}</a></th>
+          <th aria-label="Component"><a href="/certification/component/{{ component.id }}">{{ component.vendor_name }} {{ component.model}}</a></th>
           {% for lts_release, details in component.lts_releases.items() %}
```
Comment (carkod) [inline templates/certification/model-details.html:137]: Space after `component.model`?

**Comment by bethcollins92** [comment]
Thanks @carkod - made those changes ready for re-review

**Review by carkod — APPROVED** [review]
LGTM

**Pull request merged**